src/actions/ServiceGraphActions.ts

```diff
@@ -0,0 +1,16 @@
+// Action Creators allow us to create typesafe utilities for dispatching actions
+import { createAction } from 'typesafe-actions';
+
+export enum ServiceGraphActionsType {
+  TOGGLE_GRAPH_NODE_LABEL = 'TOGGLE_GRAPH_NODE_LABEL',
+  TOGGLE_GRAPH_EDGE_LABEL = 'TOGGLE_GRAPH_EDGE_LABEL'
+}
+
+export const serviceGraphActions = {
+  toggleGraphNodeLabel: createAction(ServiceGraphActionsType.TOGGLE_GRAPH_NODE_LABEL),
+  toggleGraphEdgeLabel: createAction(ServiceGraphActionsType.TOGGLE_GRAPH_EDGE_LABEL)
+  // add: createAction('ADD', (amount: number) => ({
```

Comment:
Is there a reason this dead code is here?  Not a blocker.

Reply:
It is as an example to someone building an action creator that needs arguments. I'll delete with next PR.

Reply:
don't delete it - just add some additional comments around it explaining how to do what you are trying to show. Documenting code examples for someone else is nice (especially when that "someone else" might be me).

Reply:
ok. I know trying to find that line of code again might cost some time otherwise. That is why I left it in there originally.